frontend/src/views/BannerAnnouncement.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div :class="[`${bgColor}`]">
+    <div
+      class="mx-auto py-1 px-3 w-full flex flex-row items-center justify-center flex-wrap"
+    >
+      <div class="py-1 px-3 font-medium text-white flex flex-row items-center">
+        <p>{{ announcement }}</p>
+      </div>
+      <div
+        v-if="extraDetails.length > 0"
+        class="item-center py-1 px-3 font-medium text-white truncate"
+      >
+        <a :href="extraDetails" target="_blank" class="text-center underline"
+          >See More</a
```

Comment:
Let's choose an icon from https://heroicons.com/ instead of using text. Text requires localization and a proper icon here should be enough.